tests/unit/processor/hyperscan_resolver/test_hyperscan_resolver_rule.py

```diff
@@ -108,8 +108,8 @@ def fixture_specific_rule_with_resolve_file_definition():
                     "resolve_list": {
                         "to_resolve": "resolved",
                         "other_to_resolve": "other_resolved",
-                        "store_db_persistent": True,
                     },
+                    "store_db_persistent": True,
                 },
```

Comment:
Would this be a valid rule or is it just a test case for an invalid rule?

Reply:
I have no clue, because I didn`t write this rule